rows.go

```diff
@@ -32,6 +32,7 @@ type Row struct {
 // Cell represents the data in a single cell as a consumable format.
 type Cell struct {
 	Column string // E.G   A, B, C
+	Index  int
```

Comment:
I'm not a big fan of the name here - although I can definitely see the benefit in holding the Index of a column. 

Could I suggest we turn this into a `ColumnIndex` field instead. 

For consistency, we should probably have a `RowIndex` too (which is the same as `Row - 1`).